windows-release/stage-build.yml

```diff
@@ -52,7 +55,12 @@ jobs:
         Platform: x86
         Configuration: Release
         _HostPython: .\python
-        ExtraOptions: ''
+        ${{ if ne(parameters.DoJIT, 'true') }}:
+          ExtraOptions: ''
+        ${{ elseif ne(parameters.DoJITEnabled, 'true') }}:
+          ExtraOptions: '--experimental-jit-off'
+        ${{ else }}:
+          ExtraOptions: '--experimental-jit'
```

Comment:
I don't love the repetition of this. Two options you can try:
* add an "ExtraBuildOptions" parameter to this template and put it _unconditionally_ in all of these, then have the condition in `azure-pipelines` to choose which of these to pass.
* add a `variables` block to each job in this file with an `ExtraOptions2`, and update the build steps file to use both

Possibly some combination of the two approaches is best. The main thing I want to optimise for is only seeing the two options (i.e. `'--experimental-jit'`) once (each), since they're global across the whole matrix.